ida-controller.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
As per the discussion, looks like we need changes in eSignet helm chart to respond with content of this file and IDA needs to use the domain name itself as the issuer to be in sync with that we show as identitfier in the openid vci configuration